types/tuya-panel-kit/index.d.ts

```diff
@@ -0,0 +1,2320 @@
+// Type definitions for tuya-panel-kit 2.0
+// Project: https://github.com/TuyaInc/tuya-panel-kit#readme
+// Definitions by: youngjuning <https://github.com/youngjuning>
+//                 ShinyLeee <https://github.com/ShinyLeee>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.8
+
+import * as React from 'react';
+import {
+    StyleProp,
+    ViewStyle,
+    TextStyle,
+    TextProps,
+    TouchableOpacityProps,
+    ScrollViewProps,
+    FlatListProps,
+    ViewPagerAndroidProps,
+    NativeSyntheticEvent,
+    NativeScrollEvent,
+    ViewProps,
+    TextInputProps,
+    ImageStyle,
+    ImageSourcePropType,
+    GestureResponderEvent,
+    PickerIOSProps,
+    PickerProps,
+    ModalProps as ModalNativeProps,
+    SectionListProps,
+    ListRenderItem,
+} from 'react-native';
+import { ParamListBase, RouteProp } from './@react-navigation/native';
+import { StackNavigationOptions } from './@react-navigation/stack';
+import { GlobalTheme, StopsProps, LinearGradientBackground, RadialGradientBackground, BackgroundProps } from './theme';
+
+export { GlobalTheme } from './theme';
+
+// tslint:disable-next-line  no-declare-current-package
+declare module 'tuya-panel-kit' {
```

Comment:
Why have an export, plus a nested `declare module`? Does the nested declare module allow use of its contents without importing them?

Reply:
I have removed it.:)